### ISSUE TITLE
Upgrade from C127 -> C128 on Win x64 via 1.70.x - Nightly (Checking C128)

### ISSUE THREAD

**Issue by LaurenWags** [brave/brave-browser#40306]
As per process, QA runs through the following cases to ensure that the major chromium bump that's about to get merged into `master` doesn't regress the `Nightly` channel. Once the major chromium bump is merged into `master`, QA will run through a full manual pass on `Windows` & `Android`.

### Startup & Components

- [x] Verify that Brave is only contacting `*.brave.com` endpoints on first launch using either `Charles Proxy`, `Fiddler`, `Wireshark` or `LittleSnitch` (or a similar application)
  - [x] Verify that opening a NTP doesn't trigger any outbound connections related to widgets without user interaction
- [x] Remove the following component folders and ensure that they're being re-downloaded after restarting the browser:
  - [x] `afalakplffnnnlkncjhbmahjfjhmlkal`: `AutoplayWhitelist.dat`, `ExtensionWhitelist.dat`, `ReferrerWhitelist.json` and `Greaselion.json`
  - [x] `CertificateRevocation`
  - [x] `gccbbckogglekeggclmmekihdgdpdgoe`: (Sponsored New Tab Images)
  - [x] `gkboaolpopklhgplhaaiboijnklogmbc`: Brave Ad Block List Catalog
  - [x] `iodkpdagapdfkphljnddpjlldadblomo`: Brave Ad Block Updater
  - [x] `mfddibmblmbccpadfndgakiopmmhebop`: Brave Ad Block List Catalog
  - [x] `oofiananboodjbbmdelgdommihjbkfag`: HTTPSE
  - [x] `Safe Browsing`
- [x] Restart the browser, load `brave://components`, wait for 8 mins and verify that no component shows any errors

**Note:** Always double check `brave://components` to make sure there's no errors/missing version numbers

### Upgrade

- [x] Make sure that data from the last version appears in the new version OK
- [x] Ensure that `brave://version` lists the expected Brave & Chromium versions
- [x] With data from the last version, verify that:
  - [x] Bookmarks on the bookmark toolbar and bookmark folders can be opened
  - [x] Cookies are preserved
  - [x] Installed extensions are retained and work correctly
  - [x] Opened tabs can be reloaded
  - [x] Stored passwords are preserved
  - [x] Sync chain created in previous version is retained
  - [x] Social media-blocking buttons changes are retained
  - [x] Custom filters under brave://settings/shields/filters are retained
  - [x] Custom lists under brave://settings/shields/filters are retained
  - [x] Rewards
    - [x] BAT balance is retained
    - [x] Auto-contribute list is retained
    - [x] Both Tips and Monthly Contributions are retained
    - [x] Panel transactions list is retained
    - [x] Changes to rewards settings are retained
    - [x] Ensure that Auto Contribute is not being enabled when upgrading to a new version if AC was disabled
  - [x] Ads
    - [x] Both `Estimated pending rewards` & `Ad notifications received this month` are retained
    - [x] Changes to ads settings are retained
    - [x] Ensure that ads are not being enabled when upgrading to a new version if they were disabled
    - [x] Ensure that ads are not disabled when upgrading to a new version if they were enabled

**Comment by MadhaviSeelam** [comment]
Verification `In progress` using

```
Brave | 1.70.50 Chromium: 128.0.6613.18 (Official Build) nightly (64-bit)
-- | --
Revision | 78c1a2769e6dcf20b8daf7a2c6b9b902db7ac4b3
OS | Windows 11 Version 23H2 (Build 22631.3958)
```
<details>
<summary>C127 --> C128 on-demand upgrade check - PASSED</summary>

1. installed latest nightly `1.70.50 Chromium: 127.0.6533.88 (Official Build) nightly (64-bit) ` and closed Brave
2. created `HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\BraveSoftware\UpdateDev\url` and set `url` as the following:
  * `https://updates.brave.com/service/update2`
3. set `HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\BraveSoftware\Update\ClientState\{C6CB981E-DB30-4876-8639-109F8933582C}\ap` as `nightly` (usually `cr-bump-update-tests-x64` is used but this time around `nightly` was used)
4. relaunched Brave and visited `brave://settings/help` and ensured that new update was being pulled/downloaded
5. ensured that clicking `Relaunch` restarted the browser without any issues
6. ensured Brave was updated to `1.70.50 Chromium: 128.0.6613.18 (Official Build) nightly (64-bit)` without any issues and you can restart the browser

**`step 1`** | **`step 2`** | **`step 3`** | **`step 4`** | **`step 6`**
----------------|----------------|----------------|---------------|------------
<img width="958" alt="image" src="https://github.com/user-attachments/assets/44404d8a-8ee5-4c00-bc8e-ae610aa324e2">|<img width="958" alt="image" src="https://github.com/user-attachments/assets/1dc5d8e4-0449-4641-9588-d27e7a470525">|<img width="958" alt="image" src="https://github.com/user-attachments/assets/1ad85d42-6ba3-4636-ae22-95ae7b319077">|![image](https://github.com/user-attachments/assets/f58f0bca-2a0f-4b73-a19e-7b4a702fed44)|![image](https://github.com/user-attachments/assets/5939dd31-0ecf-422c-a8ff-358188d3878b)

</details>

<details>
<summary>C127 --> C128 background upgrade check - PASSED</summary>

1. installed latest nightly `1.70.50 Chromium: 128.0.6613.18 (Official Build) nightly (64-bit)` and closed Brave
2. created `HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\BraveSoftware\UpdateDev\url` and set `url` as the following:

-   * `https://updates.brave.com/service/update2`

3. set `HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\BraveSoftware\Update\ClientState\{C6CB981E-DB30-4876-8639-109F8933582C}\ap` as `nightly` (usually `cr-bump-update-tests-x64` is used but this time around `nightly` was used)
4. started `BraveSoftwareUpdateTaskMachineUA{B9953B68-A26F-4EB2-BE09-CF9409609028}` via `Task Scheduler`
5. waited ~30 min or so before launching Brave and ensured it was updated to `1.69.44 Chromium: 127.0.6533.17`  (removed `last checked` in `BraveSoftware\Update\` 
6. restarted Brave several times and ensured that there's no issues re-launching Brave once it was updated via the `background`

**`step 1`** | **`step 2`** | **`step 3`** | **`step 4`** | **`step 5`**
----------------|----------------|---------------|----------------|-----------
<img width="958" alt="image" src="https://github.com/user-attachments/assets/44404d8a-8ee5-4c00-bc8e-ae610aa324e2">|<img width="958" alt="image" src="https://github.com/user-attachments/assets/1dc5d8e4-0449-4641-9588-d27e7a470525">|<img width="958" alt="image" src="https://github.com/user-attachments/assets/1ad85d42-6ba3-4636-ae22-95ae7b319077">|<img width="958" alt="image" src="https://github.com/user-attachments/assets/1e13bd25-5f45-4dc8-a5a8-1895f4422d2d">|<img width="958" alt="image" src="https://github.com/user-attachments/assets/a0151905-e047-47df-9218-c2d71fbed112">
</details>

<details>
<summary>C128 --> C128 on-demand upgrade check - Pending</summary>

1. installed `1.70.50 Chromium: 127.0.6533.88 (Official Build) nightly (64-bit)  ` 
2. created `HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\BraveSoftware\UpdateDev\url` and set `url` as the following:

- * `https://updates.brave.com/service/update2`

3. set `HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\BraveSoftware\Update\ClientState\{C6CB981E-DB30-4876-8639-109F8933582C}\ap` as `nightly` (usually `cr-bump-update-tests-x64` is used but this time around `nightly` was used)
4. relaunched Brave and visited `brave://settings/help` and ensured that new update was being pulled/downloaded
5. ensured that clicking `Relaunch` restarted the browser without any issues
6. ensured Brave was updated to `1.69.44 Chromium: 127.0.6533.17` without any issues and you can restart the browser

**`step 1`** | **`step 2`** | **`step 3`** | **`step 4`** | **`step 5`**
----------------|----------------|----------------|---------------|----------

</details>

<details>
<summary>C128 --> C128 background upgrade check - Pending</summary>

1. installed `1.69.37 Chromium: 127.0.6533.17` and closed Brave
2.  created `HKEY_LOCAL_MACHINE\SOFTWARE\BraveSoftware\UpdateDev\url` and set `url` as the following:
    * `https://updates.brave.com/service/update2`
3.  set `Computer\HKEY_LOCAL_MACHINE\SOFTWARE\BraveSoftware\Update\ClientState\{C6CB981E-DB30-4876-8639-109F8933582C}\ap` as `nightly` (usually `cr-bump-update-tests-x86` is used but this time around `nightly` was used)
4. started `BraveSoftwareUpdateTaskMachineUA{44272D5D-BC46-46D4-A340-7DF87CBC1459}` via `Task Scheduler`
5. waited ~30min or so before launching Brave and ensured it was updated to `1.69.44 Chromium: 127.0.6533.17 `
6. restarted Brave several times and ensured that there's no issues re-launching Brave once it was updated via the `background

**`step 1`** | **`step 2`** | **`step 3`** | **`step 4`** | **`step 5`**  
----------------|----------------|------------|------------|--------- 

</details>

**Comment by kjozwiak** [comment]
Verification **PASSED** on `Win 10 x86` using the following build(s):

```
Brave | 1.70.53 Chromium: 128.0.6613.18 (Official Build) nightly (32-bit)
-- | --
Revision | 83a4be99f404ccc245c4f5edc4690baea5beb31c
OS | Windows 10 Version 22H2 (Build 19045.4651)
```

<details>
<summary>C127 --> C128 on-demand upgrade check - PASSED</summary>

* installed `1.70.49 Chromium: 127.0.6533.88` and closed Brave
* created `HKEY_LOCAL_MACHINE\SOFTWARE\BraveSoftware\UpdateDev\url` and set `url` as the following:
  * `https://updates.brave.com/service/update2`
* set `Computer\HKEY_LOCAL_MACHINE\SOFTWARE\BraveSoftware\Update\ClientState\{C6CB981E-DB30-4876-8639-109F8933582C}\ap` as `nightly` (usually `cr-bump-update-tests-x86` is used but this time around `nightly` was used)
* relaunched Brave and visited `brave://settings/help` and ensured that new update was being pulled/downloaded
* ensured that clicking `Relaunch` restarted the browser without any issues
* ensured Brave was updated to `1.70.53 Chromium: 128.0.6613.18` without any issues and you can restart the browser

**`Example`** | **`Example`** | **`Example`** | **`Example`** 
----------------|----------------|----------------|---------------
![image](https://github.com/user-attachments/assets/52ce1959-19c4-40c8-8048-8dab1a0435c5) | ![image](https://github.com/user-attachments/assets/fdac4f4a-219a-4cf2-8152-07a06be39bc3) | ![image](https://github.com/user-attachments/assets/dc1c84e6-ff9a-4849-9e4e-006e46e090ff) | ![image](https://github.com/user-attachments/assets/fcf0c41b-07c9-4350-a3be-f2284cf7f1fa)
</details>

<details>
<summary>C128 --> C128 on-demand upgrade check - PASSED</summary>

* installed `1.70.50 Chromium: 128.0.6613.18` via `BraveBrowserStandaloneNightlySetup32_128_1_70_50.exe` and closed Brave
* created `HKEY_LOCAL_MACHINE\SOFTWARE\BraveSoftware\UpdateDev\url` and set `url` as the following:
  * `https://updates.brave.com/service/update2`
* set `Computer\HKEY_LOCAL_MACHINE\SOFTWARE\BraveSoftware\Update\ClientState\{C6CB981E-DB30-4876-8639-109F8933582C}\ap` as `nightly` (usually `cr-bump-update-tests-x86` is used but this time around `nightly` was used)
* relaunched Brave and visited `brave://settings/help` and ensured that new update was being pulled/downloaded
* ensured that clicking `Relaunch` restarted the browser without any issues
* ensured Brave was updated to `1.70.53 Chromium: 128.0.6613.18` without any issues and you can restart the browser

**`Example`** | **`Example`** | **`Example`** | **`Example`** 
----------------|----------------|----------------|----------------
![image](https://github.com/user-attachments/assets/32f8e738-2259-4c13-931d-9d17c3230e31) | ![image](https://github.com/user-attachments/assets/4402be65-afb8-43fe-a319-cb2141b070c3) | ![image](https://github.com/user-attachments/assets/f00bedad-11aa-42bf-b2c9-5a6ed72025c0) | ![image](https://github.com/user-attachments/assets/f55ce87e-943c-4505-b213-56b7dd856d92)

`x86` delta logs from `C:\ProgramData\BraveSoftware\Update\Log\BraveUpdate.log`:

```
[08/07/24 16:18:16.439][BraveUpdate:goopdate][9568:8876][Installing][display name: Brave Nightly][app id: {C6CB981E-DB30-4876-8639-109F8933582C}][installer path: C:\Program Files\BraveSoftware\Update\Install\{D05C759B-8EE2-4E4D-AA6A-BCC7AF8FD838}\brave_installer-delta-ia32.exe][manifest args: --chrome-sxs][installer data: ][untrusted data: ][priority: 10]
[08/07/24 16:18:16.440][BraveUpdate:goopdate][9568:8876][Running installer][C:\Program Files\BraveSoftware\Update\Install\{D05C759B-8EE2-4E4D-AA6A-BCC7AF8FD838}\brave_installer-delta-ia32.exe][--chrome-sxs][{C6CB981E-DB30-4876-8639-109F8933582C}]
[08/07/24 16:18:25.926][BraveUpdate:goopdate][9568:8876][InstallerResult][{C6CB981E-DB30-4876-8639-109F8933582C}][0]
[08/07/24 16:18:25.926][BraveUpdate:goopdate][9568:8876][InstallApp returned][0x0][{C6CB981E-DB30-4876-8639-109F8933582C}][type:1][code: 30][][]
[08/07/24 16:18:25.988][BraveUpdate:goopdate][9568:8876][Started process][11384]
[08/07/24 16:18:25.991][BraveUpdate:goopdate][9568:8876][LogEventHelper][4][12][1][[
Application update/install.
pid=9568, ver=1.3.361.151, machine=1, extern=1, private
App={C6CB981E-DB30-4876-8639-109F8933582C}, Ver=128.1.70.50, Download result=0x00000000
App={C6CB981E-DB30-4876-8639-109F8933582C}, Ver=128.1.70.50, Install result=0x00000000
]][]
[08/07/24 16:18:25.995][BraveUpdate:goopdate][9568:8876][Started process][8008]
[08/07/24 16:18:25.997][BraveUpdate:goopdate][9252:4444][DllEntry exit][0x00000000]
[08/07/24 16:18:25.996][BraveCrashHandler][11384:2892][ConfigManager::LoadGroupPolicies][Machine is not Enterprise Managed]
[08/07/24 16:18:25.997][BraveCrashHandler][11384:2892][ConfigManager::LoadGroupPolicies][No Group Policies found under key][HKLM\Software\Policies\BraveSoftware\Update\]
[08/07/24 16:18:25.997][BraveCrashHandler][11384:2892][OmahaPolicyManager::set_policy][Group Policy][[CachedOmahaPolicy][is_initialized][0][is_managed][0][auto_update_check_period_minutes][-1][download_preference][][cache_size_limit][-1][cache_life_limit][-1][updates_suppressed][-1][-1][-1][proxy_mode][][proxy_server][][proxy_pac_url][][install_default][-1][update_default][-1]]
[08/07/24 16:18:26.000][BraveCrashHandler][11384:2892][CrashHandler::RunUntilShutdown]
[08/07/24 16:18:26.009][BraveUpdate:goopdate][8008:7920][DllEntry]["C:\Program Files\BraveSoftware\Update\BraveUpdate.exe" /ping PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0iVVRGLTgiPz48cmVxdWVzdCBwcm90b2NvbD0iMy4wIiB1cGRhdGVyPSJPbWFoYSIgdXBkYXRlcnZlcnNpb249IjEuMy4zNjEuMTUxIiBzaGVsbF92ZXJzaW9uPSIxLjMuMzYxLjE1MSIgaXNtYWNoaW5lPSIxIiBzZXNzaW9uaWQ9Ins0Qzk1NUY2Ri1BMTYxLTQzREEtOUJGNS1FQ0RBNTU1QTE2NEF9IiB1c2VyaWQ9Ins2MTc1OUE3RS1CNDlDLTQ5NkQtQjZDNi02OEVENUY5MTI0QUJ9IiBpbnN0YWxsc291cmNlPSJ1cGRhdGUzd2ViLW9uZGVtYW5kIiB0ZXN0c291cmNlPSJhdXRvIiByZXF1ZXN0aWQ9InswMTcyRUE5RC04NEUyLTRCRTYtOUE3QS1CMjZFRDBFMDU3NkV9IiBkZWR1cD0iY3IiIGRvbWFpbmpvaW5lZD0iMCI-PGh3IHBoeXNtZW1vcnk9IjMiIHNzZT0iMSIgc3NlMj0iMSIgc3NlMz0iMSIgc3NzZTM9IjEiIHNzZTQxPSIxIiBzc2U0Mj0iMSIgYXZ4PSIxIi8-PG9zIHBsYXRmb3JtPSJ3aW4iIHZlcnNpb249IjEwLjAuMTkwNDUuNDY1MSIgc3A9IiIgYXJjaD0ieDg2Ii8-PGFwcCBhcHBpZD0ie0M2Q0I5ODFFLURCMzAtNDg3Ni04NjM5LTEwOUY4OTMzNTgyQ30iIHZlcnNpb249IjEyOC4xLjcwLjUwIiBuZXh0dmVyc2lvbj0iMTI4LjEuNzAuNTMiIGFwPSJuaWdodGx5IiBsYW5nPSIiIGJyYW5kPSJHR0xTIiBjbGllbnQ9IiIgaW5zdGFsbGFnZT0iMCIgaW5zdGFsbGRhdGU9IjY0MjYiPjxldmVudCBldmVudHR5cGU9IjEyIiBldmVudHJlc3VsdD0iMSIgZXJyb3Jjb2RlPSIwIiBleHRyYWNvZGUxPSIwIi8-PGV2ZW50IGV2ZW50dHlwZT0iMTMiIGV2ZW50cmVzdWx0PSIxIiBlcnJvcmNvZGU9IjAiIGV4dHJhY29kZTE9IjAiLz48ZXZlbnQgZXZlbnR0eXBlPSIxNCIgZXZlbnRyZXN1bHQ9IjEiIGVycm9yY29kZT0iMCIgZXh0cmFjb2RlMT0iMCIgZG93bmxvYWRlcj0iYml0cyIgdXJsPSJodHRwczovL3VwZGF0ZXMtY2RuLmJyYXZlLmNvbS9kZWx0YS9CcmF2ZS1Ccm93c2VyLU5pZ2h0bHkvbmlnaHRseS93aW4vMTI4LjEuNzAuNTMvMTI4LjEuNzAuNTAveDg2L2JyYXZlX2luc3RhbGxlci1kZWx0YS1pYTMyLmV4ZSIgZG93bmxvYWRlZD0iMzU3MDIwMCIgdG90YWw9IjM1NzAyMDAiIGRvd25sb2FkX3RpbWVfbXM9IjE0MDQiLz48ZXZlbnQgZXZlbnR0eXBlPSIxNCIgZXZlbnRyZXN1bHQ9IjEiIGVycm9yY29kZT0iMCIgZXh0cmFjb2RlMT0iMCIvPjxldmVudCBldmVudHR5cGU9IjE1IiBldmVudHJlc3VsdD0iMSIgZXJyb3Jjb2RlPSIwIiBleHRyYWNvZGUxPSIwIi8-PGV2ZW50IGV2ZW50dHlwZT0iMyIgZXZlbnRyZXN1bHQ9IjEiIGVycm9yY29kZT0iMCIgZXh0cmFjb2RlMT0iMTk2NjA5IiBzb3VyY2VfdXJsX2luZGV4PSIwIiB1cGRhdGVfY2hlY2tfdGltZV9tcz0iNDA0IiBkb3dubG9hZF90aW1lX21zPSIxNDc3IiBkb3dubG9hZGVkPSIzNTcwMjAwIiB0b3RhbD0iMzU3MDIwMCIgaW5zdGFsbF90aW1lX21zPSI5NDg3Ii8-PC9hcHA-PC9yZXF1ZXN0Pg]
[08/07/24 16:18:26.010][BraveUpdate:goopdate][8008:7920][ConfigManager::LoadGroupPolicies][Machine is not Enterprise Managed]
[08/07/24 16:18:26.010][BraveUpdate:goopdate][8008:7920][ConfigManager::LoadGroupPolicies][No Group Policies found under key][HKLM\Software\Policies\BraveSoftware\Update\]
[08/07/24 16:18:26.010][BraveUpdate:goopdate][8008:7920][OmahaPolicyManager::set_policy][Group Policy][[CachedOmahaPolicy][is_initialized][0][is_managed][0][auto_update_check_period_minutes][-1][download_preference][][cache_size_limit][-1][cache_life_limit][-1][updates_suppressed][-1][-1][-1][proxy_mode][][proxy_server][][proxy_pac_url][][install_default][-1][update_default][-1]]
[08/07/24 16:18:26.011][BraveUpdate:goopdate][8008:7920][C:\Program Files\BraveSoftware\Update\1.3.361.151\goopdate.dll][version 1.3.361.151][opt][dev]
[08/07/24 16:18:26.012][BraveUpdate:goopdate][8008:7920][is machine: 1]
[08/07/24 16:18:26.547][BraveUpdate:goopdate][8008:7920][DllEntry exit][0x00000000]
[08/07/24 16:18:26.547][BraveUpdate:goopdate][9568:11256][DllEntry exit][0x00000000]
```
</details>

<details>
<summary>C127 --> C128 background upgrade check - PASSED</summary>

* installed `1.70.49 Chromium: 127.0.6533.88` and closed Brave
* created `HKEY_LOCAL_MACHINE\SOFTWARE\BraveSoftware\UpdateDev\url` and set `url` as the following:
  * `https://updates.brave.com/service/update2`
* set `Computer\HKEY_LOCAL_MACHINE\SOFTWARE\BraveSoftware\Update\ClientState\{C6CB981E-DB30-4876-8639-109F8933582C}\ap` as `nightly` (usually `cr-bump-update-tests-x86` is used but this time around `nightly` was used)
* started `BraveSoftwareUpdateTaskMachineUA{C652585D-72D4-47C2-ADD7-E4154C822BD1}` via `Task Scheduler`
* waited ~30min or so before launching Brave and ensured it was updated to `1.70.53 Chromium: 128.0.6613.18`
* restarted Brave several times and ensured that there's no issues re-launching Brave once it was updated via the `background`

**`Example`** | **`Example`** | **`Example`** | **`Example`**
----------------|----------------|---------------|-----------------
![image](https://github.com/user-attachments/assets/d389c3b8-5bc1-4ada-b06e-eeae1d1c41e0) | ![image](https://github.com/user-attachments/assets/34615fea-5267-49e4-93d5-64e38c111669) | ![image](https://github.com/user-attachments/assets/61a7ce3d-b981-4399-b58f-01e949ca87b8) | ![image](https://github.com/user-attachments/assets/bd55d1a6-d21a-4824-aea1-5c268f9d8cf0)
</details>

<details>
<summary>C128 --> C128 background upgrade check - PASSED</summary>

* installed `1.70.50 Chromium: 128.0.6613.18` via `BraveBrowserStandaloneNightlySetup32_128_1_70_50.exe` and closed Brave
* created `HKEY_LOCAL_MACHINE\SOFTWARE\BraveSoftware\UpdateDev\url` and set `url` as the following:
  * `https://updates.brave.com/service/update2`
* set `Computer\HKEY_LOCAL_MACHINE\SOFTWARE\BraveSoftware\Update\ClientState\{C6CB981E-DB30-4876-8639-109F8933582C}\ap` as `nightly` (usually `cr-bump-update-tests-x86` is used but this time around `nightly` was used)
* started `BraveSoftwareUpdateTaskMachineUA{255700BD-7CBB-476F-95DB-05324065749A}` via `Task Scheduler`
* waited ~30min or so before launching Brave and ensured it was updated to `1.69.44 Chromium: 127.0.6533.17`
* restarted Brave several times and ensured that there's no issues re-launching Brave once it was updated via the `background

**`Example`** | **`Example`** | **`Example`** | **`Example`**
----------------|----------------|----------------|----------------
![image](https://github.com/user-attachments/assets/7ada5208-b2ac-499a-809d-56771715ac19) | ![image](https://github.com/user-attachments/assets/e539ee4d-0a48-4095-97d9-cbba8b5ede24) | ![image](https://github.com/user-attachments/assets/c0301379-f65e-403e-9ae3-b89247a7e9cb) | ![image](https://github.com/user-attachments/assets/6e34ef5b-8b3d-4a08-8b5c-18656468f9f2)

`x86` delta logs from `C:\ProgramData\BraveSoftware\Update\Log\BraveUpdate.log`:

```
[08/07/24 16:36:57.669][BraveUpdate:goopdate][2216:6480][Installing][display name: Brave Nightly][app id: {C6CB981E-DB30-4876-8639-109F8933582C}][installer path: C:\Program Files\BraveSoftware\Update\Install\{46E78A45-971A-4C7F-813E-893DB5129811}\brave_installer-delta-ia32.exe][manifest args: --chrome-sxs][installer data: ][untrusted data: ][priority: 0]
[08/07/24 16:36:57.669][BraveUpdate:goopdate][2216:6480][Running installer][C:\Program Files\BraveSoftware\Update\Install\{46E78A45-971A-4C7F-813E-893DB5129811}\brave_installer-delta-ia32.exe][--chrome-sxs][{C6CB981E-DB30-4876-8639-109F8933582C}]
[08/07/24 16:37:05.166][BraveUpdate:goopdate][2216:6480][InstallerResult][{C6CB981E-DB30-4876-8639-109F8933582C}][0]
[08/07/24 16:37:05.166][BraveUpdate:goopdate][2216:6480][InstallApp returned][0x0][{C6CB981E-DB30-4876-8639-109F8933582C}][type:1][code: 2][]["C:\Program Files\BraveSoftware\Brave-Browser-Nightly\Application\brave.exe" --from-installer]
[08/07/24 16:37:05.229][BraveUpdate:goopdate][2216:6480][Started process][1852]
[08/07/24 16:37:05.229][BraveUpdate:goopdate][2216:6480][LogEventHelper][4][12][1][[
Application update/install.
pid=2216, ver=1.3.361.151, machine=1, extern=1, private
App={C6CB981E-DB30-4876-8639-109F8933582C}, Ver=128.1.70.50, Download result=0x00000000
App={C6CB981E-DB30-4876-8639-109F8933582C}, Ver=128.1.70.50, Install result=0x00000000
]][]
[08/07/24 16:37:05.229][BraveCrashHandler][1852:3584][ConfigManager::LoadGroupPolicies][Machine is not Enterprise Managed]
[08/07/24 16:37:05.229][BraveCrashHandler][1852:3584][ConfigManager::LoadGroupPolicies][No Group Policies found under key][HKLM\Software\Policies\BraveSoftware\Update\]
[08/07/24 16:37:05.229][BraveCrashHandler][1852:3584][OmahaPolicyManager::set_policy][Group Policy][[CachedOmahaPolicy][is_initialized][0][is_managed][0][auto_update_check_period_minutes][-1][download_preference][][cache_size_limit][-1][cache_life_limit][-1][updates_suppressed][-1][-1][-1][proxy_mode][][proxy_server][][proxy_pac_url][][install_default][-1][update_default][-1]]
[08/07/24 16:37:05.229][BraveCrashHandler][1852:3584][CrashHandler::RunUntilShutdown]
[08/07/24 16:37:05.260][BraveUpdate:goopdate][9412:8008][DllEntry exit][0x00000000]
[08/07/24 16:37:05.260][BraveUpdate:goopdate][2216:8984][Started process][7140]
[08/07/24 16:37:05.260][BraveUpdate:goopdate][7140:4672][DllEntry]["C:\Program Files\BraveSoftware\Update\BraveUpdate.exe" /ping PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0iVVRGLTgiPz48cmVxdWVzdCBwcm90b2NvbD0iMy4wIiB1cGRhdGVyPSJPbWFoYSIgdXBkYXRlcnZlcnNpb249IjEuMy4zNjEuMTUxIiBzaGVsbF92ZXJzaW9uPSIxLjMuMzYxLjE1MSIgaXNtYWNoaW5lPSIxIiBzZXNzaW9uaWQ9Ins5MkVGQjJFRi1GQjZCLTQ1MTMtOTYyQS03NEI3MDFCOTIwNTh9IiB1c2VyaWQ9InswNTNGNzk3Qi02QTQ4LTQxRkUtQkFENC1FNjUyMUYyQ0Q5OUV9IiBpbnN0YWxsc291cmNlPSJzY2hlZHVsZXIiIHRlc3Rzb3VyY2U9ImF1dG8iIHJlcXVlc3RpZD0ie0VDODE1NTkyLThBRkQtNEQyRC05NEE1LTU4RjRCMUJCMkIwRn0iIGRlZHVwPSJjciIgZG9tYWluam9pbmVkPSIwIj48aHcgcGh5c21lbW9yeT0iMyIgc3NlPSIxIiBzc2UyPSIxIiBzc2UzPSIxIiBzc3NlMz0iMSIgc3NlNDE9IjEiIHNzZTQyPSIxIiBhdng9IjEiLz48b3MgcGxhdGZvcm09IndpbiIgdmVyc2lvbj0iMTAuMC4xOTA0NS40NjUxIiBzcD0iIiBhcmNoPSJ4ODYiLz48YXBwIGFwcGlkPSJ7QzZDQjk4MUUtREIzMC00ODc2LTg2MzktMTA5Rjg5MzM1ODJDfSIgdmVyc2lvbj0iMTI4LjEuNzAuNTAiIG5leHR2ZXJzaW9uPSIxMjguMS43MC41MyIgYXA9Im5pZ2h0bHkiIGxhbmc9IiIgYnJhbmQ9IkdHTFMiIGNsaWVudD0iIiBpbnN0YWxsYWdlPSIwIj48ZXZlbnQgZXZlbnR0eXBlPSIxMiIgZXZlbnRyZXN1bHQ9IjEiIGVycm9yY29kZT0iMCIgZXh0cmFjb2RlMT0iMCIvPjxldmVudCBldmVudHR5cGU9IjEzIiBldmVudHJlc3VsdD0iMSIgZXJyb3Jjb2RlPSIwIiBleHRyYWNvZGUxPSIwIi8-PGV2ZW50IGV2ZW50dHlwZT0iMTQiIGV2ZW50cmVzdWx0PSIxIiBlcnJvcmNvZGU9IjAiIGV4dHJhY29kZTE9IjAiIGRvd25sb2FkZXI9ImJpdHMiIHVybD0iaHR0cHM6Ly91cGRhdGVzLWNkbi5icmF2ZS5jb20vZGVsdGEvQnJhdmUtQnJvd3Nlci1OaWdodGx5L25pZ2h0bHkvd2luLzEyOC4xLjcwLjUzLzEyOC4xLjcwLjUwL3g4Ni9icmF2ZV9pbnN0YWxsZXItZGVsdGEtaWEzMi5leGUiIGRvd25sb2FkZWQ9IjM1NzAyMDAiIHRvdGFsPSIzNTcwMjAwIiBkb3dubG9hZF90aW1lX21zPSIxMzMzMiIvPjxldmVudCBldmVudHR5cGU9IjE0IiBldmVudHJlc3VsdD0iMSIgZXJyb3Jjb2RlPSIwIiBleHRyYWNvZGUxPSIwIi8-PGV2ZW50IGV2ZW50dHlwZT0iMTUiIGV2ZW50cmVzdWx0PSIxIiBlcnJvcmNvZGU9IjAiIGV4dHJhY29kZTE9IjAiLz48ZXZlbnQgZXZlbnR0eXBlPSIzIiBldmVudHJlc3VsdD0iMSIgZXJyb3Jjb2RlPSIwIiBleHRyYWNvZGUxPSIxOTY2MDkiIHNvdXJjZV91cmxfaW5kZXg9IjAiIHVwZGF0ZV9jaGVja190aW1lX21zPSI1NjIiIGRvd25sb2FkX3RpbWVfbXM9IjEzMzk0IiBkb3dubG9hZGVkPSIzNTcwMjAwIiB0b3RhbD0iMzU3MDIwMCIgaW5zdGFsbF90aW1lX21zPSI3NDk3Ii8-PC9hcHA-PC9yZXF1ZXN0Pg]
[08/07/24 16:37:05.275][BraveUpdate:goopdate][7140:4672][ConfigManager::LoadGroupPolicies][Machine is not Enterprise Managed]
[08/07/24 16:37:05.275][BraveUpdate:goopdate][7140:4672][ConfigManager::LoadGroupPolicies][No Group Policies found under key][HKLM\Software\Policies\BraveSoftware\Update\]
[08/07/24 16:37:05.275][BraveUpdate:goopdate][7140:4672][OmahaPolicyManager::set_policy][Group Policy][[CachedOmahaPolicy][is_initialized][0][is_managed][0][auto_update_check_period_minutes][-1][download_preference][][cache_size_limit][-1][cache_life_limit][-1][updates_suppressed][-1][-1][-1][proxy_mode][][proxy_server][][proxy_pac_url][][install_default][-1][update_default][-1]]
[08/07/24 16:37:05.275][BraveUpdate:goopdate][7140:4672][C:\Program Files\BraveSoftware\Update\1.3.361.151\goopdate.dll][version 1.3.361.151][opt][dev]
[08/07/24 16:37:05.275][BraveUpdate:goopdate][7140:4672][is machine: 1]
[08/07/24 16:37:05.775][BraveUpdate:goopdate][7140:4672][DllEntry exit][0x00000000]
[08/07/24 16:37:05.775][BraveUpdate:goopdate][2216:11432][DllEntry exit][0x00000000]
```
</details>
